heareval/tasks/util/audio.py

```diff
@@ -47,21 +49,23 @@ def convert_to_mono_wav(in_file: str, out_file: str):
 
 
 def resample_wav(in_file: str, out_file: str, out_sr: int):
-    # TODO: Don't do anything if we are already the right sample rate.
```

Comment:
I don't think this should be responsible for handling that. But this is a good idea to implement in the resample task perhaps? Would involve knowing the sample rate beforehand.

Reply:
Can you explain your comment?

Reply:
Ya, my thought was that this function should remain a straightforward as possible and just run the resampling command when called. i.e, it shouldn't also be responsible for checking to make sure that it needs to resample. Part of the complication is that this function performs a resample and then saves the result in the new specified location (workdir) -- what would this function do if it doesn't need to resample? Copy the file to the new directory? Return an error? Create a symlink? 

If we want to avoid the redundancy of resampling an audio file that is already at the correct sample rate then the caller should be responsible for that. From the perspective of the luigi task it would totally make sense to skip calling this resample function and just create a symlink.